BlazorApp/wwwroot/index.html

```diff
@@ -1,4 +1,3 @@
-<!DOCTYPE html>
```

Comment:
The <!DOCTYPE html> declaration at the beginning of an HTML document is extremely important because it tells the browser how to interpret the document.

Reply:
added